sandbox/load_test_jobs.py

```
@@ -701,6 +761,135 @@ def build(_idx: int) -> Shell:
+            ol = outline.Outline(short_name, shot=common.shot, show=common.show)
+            ol.add_layer(Shell(
+                "beauty",
+                command=["/bin/echo", "rendered", "frame", "#IFRAME#"],
+                range="1-%d" % nframes,
+            ))
+            # Always paused: the marker frames would otherwise finish instantly
+            # and the job would leave the active list before you can open it.
+            outline.cuerun.launch(ol, pause=True, use_pycuerun=False)
+
+            job = _find_job(short_name)
+            layer = next(l for l in job.getLayers() if l.name() == "beauty")
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)